File: OSSA/Lab Sheets/Lab1/exe11.go
package main

import (
	"fmt"
	"os"
	"time"
)

func main() {
	go func() {
		fmt.Printf("I am child process \n")
		fmt.Printf("pid : %d\n", os.Getpid())
		time.Sleep(time.Second * 10)
	}()
	fmt.Printf("I am parent process \n")
}

File: OSSA/Lab Sheets/Lab1/exe11.c
//Orphan Process
#include <stdio.h>
int main()
{
        int id;
        if ((id = fork()) == 0)
        {
                printf("I am child process \n");
                printf("pid : %d\n", getpid());
                sleep(10);
        }
        //while the child process sleeping parent process execute.but it there is wait function parent process will not execute until child process done
        //process id of init is one.
        else
        {
                printf("I am parent process \n");
        }
}
